src/audio/google/google_rtc_audio_processing.c

```diff
@@ -663,6 +718,24 @@ static int google_rtc_audio_processing_copy(struct comp_dev *dev)
 	return 0;
 }
 
+#if CONFIG_IPC_MAJOR_4
+static int google_rtc_audio_processing_get_attribute(struct comp_dev *dev,
```

Comment:
what about set?

Reply:
this module doesn't need set. We only need to add get for IPC4 path to return base_cfg. Please check comp_mixin module
````c
static const struct comp_driver comp_mixin = {
	.uid	= SOF_RT_UUID(mixin_uuid),
	.tctx	= &mixin_tr,
	.ops	= {
		.create  = mixin_new,
		.free    = mixin_free,
		.params  = mixin_params,
		.prepare = mixin_prepare,
		.trigger = mixinout_trigger,
		.copy    = mixin_copy,
		.reset   = mixin_reset,
		.get_attribute = mixin_get_attribute,
		.set_large_config = mixin_set_large_config,
	},
};
````